changelog/7119.bugfix.rst

```diff
@@ -0,0 +1 @@
+Raise exception if ``basetemp`` argument is empty string or . or parent folder of current working directory.
```

Comment:
```suggestion
Exit with an error if the ``--basetemp`` argument is empty, the current working directory or parent directory of it.
This is done to protect against accidental data loss, as any directory passed to this argument is cleared.
```

I also would classify this as an `improvement` rather than `bugfix`, but that's up to you.